source/viz.md

```diff
@@ -405,26 +412,21 @@ visualization slightly. Note that it is totally fine to use a small number of
 visualizations to answer different aspects of the question you are trying to
 answer. We will accomplish this by using *scale*,
 another important feature of `altair` that easily transforms the different
-variables and set limits.  We scale the horizontal axis using the `alt.Scale(domain=['1990', '1993'])` by restricting the x-axis values between 1990 and 1994,
-and the vertical axis with the `alt.Scale(zero=False)` function, to not start the y-axis with zero.
+variables and set limits.
 In particular, here, we will use the `alt.Scale` function to zoom in
-on just five years of data (say, 1990-1994). The
+on just a few years of data (say, 1990-1995). The
```

Comment:
is this an error we made in the R book too?

Reply:
No, it is fine in the R book, but Altair will switch to include the name of the months if we only do 90-94 which is why I extended it with one year

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/c996b79d-c28b-469b-8507-6a32f83596aa)

versus

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/280f76cb-46b2-44f9-8a78-3e91591ef97c)

Reply:
good catch!